sbt-mode-comint.el

```diff
@@ -184,14 +184,14 @@ what `sbt:move-marker-before-prompt-filter` did."
 (defun sbt:scala-escape-string (str)
   (mapconcat 'sbt:scala-escape-char str ""))
 
-(defconst sbt:completions-regex "^\\[completions\\] \\(.*\\)
?$")
+(defconst sbt:completions-regex "^\\[completions\\] \\(.*\\)
```

Comment:
Didn't even intend to do that change. I'm baffled, where did that ^M come from and where did it go?